client/src/components/Checkout/checkout.js

```diff
@@ -44,53 +45,57 @@ function goBack() {
   window.location.href="/browse";
 }
 
+const DEFAULT_ITEM = {
+  title: "",
+  desc: "",
+  lngLat: [0, 0],
+  imgUrl: "",
+  id: "",
+};
+
```

Comment:
I like your use of caps to distinguish this object. I might consider updating my code to replicate this.